### PR TITLE
flake: Update lock file.

### PR DESCRIPTION
**Pull request by drupol**

### CODE REVIEW

**Comment by drupol** [comment]
@jtojnar Do you think thin we can discuss this?

**Comment by jtojnar** [comment]
Sorry, I missed the notification for some reason. This fails for the same reason the automated update does, see https://github.com/fossar/nix-phps/pull/34#issuecomment-873587047.

**Comment by drupol** [comment]
Yep, just saw. This is beyond my knowledge by far, sadly.

**Comment by drupol** [comment]
If you have any pointers on how we could fix this, please let me know.

**Comment by jtojnar** [comment]
Well, https://github.com/fossar/nix-phps/blob/49fea59ae5ae634ee8b38e89ddd22b3dd9f49176/pkgs/package-overrides.nix#L49-L61 refers to `preCheck` attribute which does not exist anymore.

So you would remove that section and instead re-add the stuff removed in https://github.com/NixOS/nixpkgs/commit/4c424870e62a5095896b48a12e9903835f6a5e53 for PHP versions not in Nixpkgs.

**Comment by drupol** [comment]
I removed the part, but I think you should approve the running of the CI on Github.

**Comment by jtojnar** [comment]
Ugh, I thought GitHub would remember I already approved you. I guess we will have to keep approving until some of your commits are merged.

**Comment by jtojnar** [comment]
BTW, you should be able to test locally by building `php73.extensions.dom`.

**Comment by drupol** [comment]
I think the part I removed should only be applied to `php` < 7.4.

**Comment by jtojnar** [comment]
No, it needs to be removed everywhere. But you need to reintroduce what was removed in [NixOS/nixpkgs@4c42487](https://github.com/NixOS/nixpkgs/commit/4c424870e62a5095896b48a12e9903835f6a5e53) for PHP versions not in Nixpkgs.

File: pkgs/package-overrides.nix [hunk 1]
```diff
@@ -42,23 +42,23 @@ in
               url = "https://github.com/php/php-src/commit/e29922f054639a934f3077190729007896ae244c.patch";
               sha256 = "zC2QE6snAhhA7ItXgrc80WlDVczTlZEzgZsD7AS+gtw=";
             })
+          ] ++
+            lib.optionals (lib.versionOlder prev.php.version "7.4") [
+             (pkgs.fetchpatch {
+              url = "https://github.com/php/php-src/commit/4cc261aa6afca2190b1b74de39c3caa462ec6f0b.patch";
+              sha256 = "11qsdiwj1zmpfc2pgh6nr0sn7qa1nyjg4jwf69cgwnd57qfjcy4k";
+              excludes = [ "ext/dom/tests/bug43364.phpt" "ext/dom/tests/bug80268.phpt" ];
+            })
```
Comment (jtojnar) [inline pkgs/package-overrides.nix:51]: The indent does not match here, should be 2 spaces.

Comment (drupol) [reply pkgs/package-overrides.nix:51]: oops. Going to fix that.

Comment (jtojnar) [reply pkgs/package-overrides.nix:51]: Also would be nice to have excludes one per line.

Comment (jtojnar) [reply pkgs/package-overrides.nix:54]: Still the closing parens look misaligned.

**Comment by jtojnar** [comment]
Thanks.

**Pull request merged**